keda/templates/12-keda-deployment.yaml

```diff
@@ -100,6 +100,10 @@ spec:
             - name: KEDA_HTTP_DEFAULT_TIMEOUT
               value: {{ .Values.http.timeout | quote }}
             {{- end }}
+            {{- if .Values.http.disableKeepAlive }}
```

Comment:
Let's use this approach which follows our typical way of configuring flags
```suggestion
            {{- if .Values.http.keepAlive.enabled }}
```

Reply:
Remember during the change that this requires changes in docs and code too ;)

Reply:
@tomkerkhove @JorTurFer , as http transports is "DisableKeepAlives", so i transform the value in helm, please review again, thanks.
<img width="941" alt="image" src="https://user-images.githubusercontent.com/13001038/204690509-8b8d07d2-ecda-4009-8e44-c242bd56a867.png">